content/CORE/CORETutorials/Services/ConfiguringLLDP.md

```diff
@@ -16,4 +19,4 @@ Select **Interface Description** and enter a **Country Code**. The location of t
 
 Turn on the LLDP service on.
 
-See [LLDP Screen]({{< relref "/CORE/UIReference/Services/LLDPScreen.md" >}}) for more information on the settings.
+{{< taglist tag="corelldp" limit="10" title="Related Articles" >}}
```

Comment:
the title="related articles" is not needed in this shortcode as it is there by default.
```suggestion
{{< taglist tag="corelldp" limit="10" >}}
```

Reply:
Deleted taglist title parameter for corelldp.